### DISCUSSION TITLE
How to allow nx serve/build to continue building a Node app, regardless of whether there are TypeScript errors

### DISCUSSION THREAD

**Discussion by hi2u** [nrwl/nx#17282] [General]
I have Node apps generated with commands like:

```
nx g @nx/node:application mynodeappname
```

I pick framework=none

Up to Nx v16.0.0 (and including)... `nx serve` + `nx build` would still work, even if there were TypeScript errors.

After upgrading Nx v16.0.0 -> v.16.2.1... this no longer works.  Both `nx serve` + `nx build` refuse to continue if there are any TypeScript errors at all.  This is a huge regression in productivity for me, as I need to be able to run code, even with TypeScript errors temporarily... as it's very common to have them part-way through refactoring, and I do a lot of type generation etc.

How do I get the old  behavior back so that I can still compile? (as I could on v15.* and v16.0.0)

Is there a `project.json` setting I can use or something?

**Comment by melonges** [comment]
i have the same problem

**Reply by hi2u** [reply]
This seems to be ok again for me on Nx v16.6.0.

I can successfully compile my Node app with TS errors with commands:

- ✔️ `nx serve mynodeapp`
- ✔️ `nx build mynodeapp --watch`

Although this still refuses to build:
- ❌ `nx build mynodeapp`

Are any of these working for you now @melonges ?

**Reply by melonges** [reply]
thanks a lot
it was easier for me to correct my ts errors or use ts-ignore
